Core/include/Acts/Propagator/StepperExtensionList.hpp

```diff
@@ -44,8 +43,6 @@ struct StepperExtensionList : private detail::Extendable<extensions...> {
   // Access to all extensions
   using detail::Extendable<extensions...>::tuple;
```

Comment:
I know this is not really part of the PR, but if you're already modifying this, could you consider giving this a more descriptive name. I think 

```cpp
using extensionTuple = detail::Extencable<extensions...>::tuple;
```

would work, for example.

Reply:
Hmm I don't think this works (and also tried it), since `using detail::Extencable<extensions...>::tuple` is a function-using, whereas what you're suggesting is a typename-alias... 
What one could do is redirect the `tuple()` function using a wrapper-function... But not sure if this is worth the effort.

Reply:
Yeah no ok.